lib/src/sql/statements/show.rs

```diff
@@ -0,0 +1,206 @@
+use crate::ctx::Context;
+use crate::dbs::Level;
+use crate::dbs::Options;
+use crate::err::Error;
+use crate::sql::comment::shouldbespace;
+use crate::sql::error::IResult;
+use crate::sql::table::{table, Table};
+use crate::sql::value::Value;
+use derive::Store;
+use nom::branch::alt;
+use nom::bytes::complete::tag_no_case;
+use nom::combinator::map;
+use nom::combinator::opt;
+use nom::combinator::recognize;
+use nom::sequence::preceded;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+// ShowStatement is used to show changes in a table or database via
+// the SHOW CHANGES statement.
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize, Store, Hash)]
+pub struct ShowStatement {
+	pub table: Option<String>,
+	pub since: Option<u64>,
+	pub limit: Option<u32>,
+}
+
+impl ShowStatement {
+	/// Process this type returning a computed simple Value
+	pub(crate) async fn compute(&self, ctx: &Context<'_>, opt: &Options) -> Result<Value, Error> {
+		// Selected DB?
+		opt.needs(Level::Db)?;
+		// Allowed to run?
+		opt.check(Level::Db)?;
+		// Allowed to run?
+		opt.check(Level::No)?;
```

Comment:
this second check is not really useful, because if line 34 succeed, this will always succeed.

Reply:
Thanks for pointing it out! Removed L35-36!